internal/gengapic/client_init.go

```diff
@@ -160,7 +160,13 @@ func (g *generator) clientOptions(serv *descriptor.ServiceDescriptorProto, servN
 				p("gax.WithRetry(func() gax.Retryer {")
 				p("  return gax.OnCodes([]codes.Code{")
 				for _, c := range rp.GetRetryableStatusCodes() {
-					p("    codes.%s,", snakeToCamel(c.String()))
+					// Go uses the American-English spelling with a single "L"
+					code := c.String()
+					if code == "CANCELLED" {
+						code = "CANCELED"
```

Comment:
Just wondering, can we use that `grpc.status` enum directly without having a string constant?  In JavaScript, we can:

```
> grpc.status.CANCELLED
1
```

Reply:
Great idea! Done.